Terminal.Gui/Windows/FileDialog.cs

```diff
@@ -275,6 +303,46 @@ List<string> GetFilesName (IReadOnlyList<string> files)
 			return filesName;
 		}
 
+		public bool GetValidFilesName (string files, out string result)
+		{
+			result = string.Empty;
+			if (infos?.Count == 0) {
+				return false;
+			}
+
+			var valid = true;
+			IReadOnlyList<string> filesList = new List<string> (files.Split (splitString.ToArray (), StringSplitOptions.None));
```

Comment:
I don't quite understand the splitting on ", " and string manipulation here.  Is it presentation layer only or does it control returned values from control.  Would a filename with a comma or comma space in its name cause a problem?

Reply:
That is for AllowsMultipleSelection = true.

![filedialog-splitting](https://user-images.githubusercontent.com/13117724/127938011-ed9cfc4d-b958-47dd-8e12-667fb506e40e.gif)


> Would a filename with a comma or comma space in its name cause a problem?
>

No because it will use the FilePaths.